cachi2/core/config.py

```diff
@@ -1,24 +1,24 @@
-config = None
+from typing import Optional
+
+import typer
+import yaml
+from pydantic import BaseModel
 
+config = None
 
-class Config:
-    """
-    Singleton that provides default configuration for the Cachi2 process.
 
-    All values currently need to be changed in this file.
-    """
+class Config(BaseModel):
```

Comment:
Do we want to forbid the user from setting not mapped attributes here? We can use `extra="forbid"` to do that.

e.g. https://github.com/containerbuildsystem/cachi2/blob/main/cachi2/core/models/input.py#L53